README.md

```diff
@@ -47,6 +47,14 @@ The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to po
 
 Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of your eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
 
+### Tag's Configuration
+
+The connector reads from the tag's historical logs to write to OSIsoft. For best results, enable the Logging Deadband feature. The settings for the historical log deadband are the same as the settings for the OSIsoft connector deadband.
+
+#### Deadband
+
+Deadband can be set on the flexy by changing the mode to setup and looking at the historical logging section of the settings. If you set a deadband of 1, then the tag value will need to change by a value of 1 or more before being logged. If you have logging deadband disabled (set to -1) we recommend you set the historical log's logging interval setting to something high if possible. Set the deadband to whatever your lowest unit of interest is.
```

Comment:
"changing the mode to setup" what mode? You need to be clear.

Does changing this remove the need for the postduplicatetagvalues parameter? If so that should be removed from the config, webpage, and docs.

Reply:
I guess post duplicate values isn't needed anymore. I'm adding that change to another commit